addons/l10n_mz/data/account_tax_template_data.xml

```diff
@@ -0,0 +1,358 @@
+                }),
+                (0, 0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                }),
+            ]"/>
+        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+                (0, 0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                    'minus_report_expression_ids': [ref('l10n_mz_tax_report_3_tag')],
+                }),
+                (0, 0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                }),
+            ]"/>
+    </record>
+    <record model="account.tax.template" id="vat_exempt_sale">
+        <field name="name">0% Exempt</field>
+        <field name="description">0%</field>
+        <field name="amount" eval="0"/>
+        <field name="amount_type">percent</field>
+        <field name="chart_template_id" ref="l10n_mz_chart_template"/>
+        <field name="type_tax_use">sale</field>
+        <field name="tax_group_id" ref="tax_group_vat_0"/>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+                (0, 0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                    'plus_report_expression_ids': [ref('l10n_mz_tax_report_4_tag')],
+                }),
+                (0, 0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                }),
+            ]"/>
+        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+                (0, 0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                    'minus_report_expression_ids': [ref('l10n_mz_tax_report_4_tag')],
+                }),
+                (0, 0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                }),
+            ]"/>
+    </record>
+    <record model="account.tax.template" id="vat_purch_16_fixed">
+        <field name="name">16% G. Purchase of Fixed Assets</field>
+        <field name="description">16%</field>
+        <field name="amount" eval="16"/>
+        <field name="amount_type">percent</field>
+        <field name="chart_template_id" ref="l10n_mz_chart_template"/>
+        <field name="type_tax_use">purchase</field>
+        <field name="tax_group_id" ref="tax_group_vat_16"/>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+                (0, 0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                }),
+                (0, 0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                    'account_id': ref('l10n_mz_account_44322'),
+                    'plus_report_expression_ids': [ref('l10n_mz_tax_report_5_tag')]
```

Comment:
```suggestion
                    'plus_report_expression_ids': [ref('l10n_mz_tax_report_5_tag')],
```
When you do dictionaries like this (split in several lines), you need to add a coma at the end of every line. If someone someday needs to add a value to it, he will need to add a coma and then it will "break" the git blame for no good reason.

(This applies other places in this file)